bitnami/solr/templates/exporter-deployment.yaml

```diff
@@ -0,0 +1,108 @@
+{{- if .Values.exporter.enabled }}
+---
```

Comment:
This is not needed since there is only one resource defined into this file

Reply:
But if the exporter.enabled is false, we dont need to create this. Thats why I had added it.

Reply:
I mean the `---`

Reply:
sure :)

Reply:
made the change here.